boardgame-ui/src/locales/translations.csv

```diff
@@ -127,6 +128,8 @@ sort.score.increasing,Score (increasing),
 sort.score.decreasing,Score (decreasing),
 board-games-list.add,Add board game,
 board-games-list.belongsToUser,Only mine,
+board-games-list.no-games-fitting-criteria,No board games fitting search critera,
```

Comment:
"No board game" or "No board games"?

Reply:
I think in this case the plural is OK (http://www.englishlessonsbrighton.co.uk/followed-singular-plural-noun/)

Reply:
Good to know!